api/v1alpha1/backendtrafficpolicy_types.go

```diff
@@ -74,7 +74,6 @@ type BackendTrafficPolicySpec struct {
 	// If multiple configurations are specified, the first one to match wins.
 	//
 	// +optional
-	// +notImplementedHide
 	ResponseOverride []*ResponseOverride `json:"responseOverride,omitempty"`
```

Comment:
I'm considering encapsulating the respond overriding rules within a top-level structure to make it more future-proof. This would allow us to add some global settings in the future.

```yaml
  respondOverride:
    rules:
      - match: ....
        response: ... 
      - match: ....
        response: ... 
```  

WDYT @arkodg

Reply:
can you think of any global settings ?
`rules` adds one more level of config authorship

Reply:
I don't have a concrete example in my mind yet. A possible use case could be adding a global header in the response for all the rules?

This is non-blocking though.